### ISSUE TITLE
[FEATURE] AdaHandle support

### ISSUE THREAD

**Issue by matiwinnetou** [cardano-foundation/cardano-connect-with-wallet#56]
**Is your feature request related to a problem? Please describe.**

Many Cardano Connect with Wallet integrations upon connection let a user choose ada handle. Once ada handle is chosen, the wallet connector doesn't display a stake address but ada handle, e.g. $matiwinnetou

**Describe the solution you'd like**
Add additional property to the react component so it is possible to pick adahandle and once user picks adahandle it becomes sticky.

**Describe alternatives you've considered**
Basically one can extend cardano-connect-with-wallet, fork it and provide this functionality without being in the common component.

**Comment by fabianbormann** [comment]
Hi @matiwinnetou, if I'm right, we would need to query all assets of the stake address and search for the handles from e.g. koios and then use it as the default button title (if connected). https://docs.adahandle.com/quick-start

A feasible and lightweight solution would be to add a connectedLabel prop (in contrast to [that prop](https://github.com/cardano-foundation/cardano-connect-with-wallet/blob/main/react/src/components/ConnectWalletButton/ConnectWalletButton.tsx#L31)) so that a developer would be able to fetch the adahandle via koios/blockfrost, show a selection dialog and then he/she would be able to do:

```js
<ConnectWalletButton connectedLabel={adahandle} />
```

...

A more complex alternative would be to introduce a prop `useAdaHandle` and implement the fetch- and display-logic inside the component. I don't know if this would bring to much load to koios or if we would need to introduce a fetchAssetsUrl, so that devs could use their own blockfrost api key inside a backend service 🤔 .. what do you think about it?

**Comment by fabianbormann** [comment]
As discussed with @satran004 we will now work on a service to resolve adahandles using yaci-store and bring that open source and 24/7 up in the cf environment allowing also community projects to use it 😊 .. I will link the project and it's status here once it's released in v0.0.1 😉

**Comment by fabianbormann** [comment]
Almost done https://github.com/cardano-foundation/adahandle-resolver 🙌
